tools/depends/target/cmakebuildsys/Makefile

```diff
@@ -0,0 +1,16 @@
+-include ../../Makefile.include
+
+VERSION.TXT := $(XBMCROOT)/version.txt
+APP_NAME=$(shell awk '/APP_NAME/ {print tolower($$2)}' $(VERSION.TXT))
+
+all: $(XBMCROOT)/lib$(APP_NAME).so
+
+$(XBMCROOT)/lib$(APP_NAME).so:
```

Comment:
Ah true, do you have a suggestion how it should be done? Depending on what CMake generator we use it might generate different output. (with GNU Makefiles we build ./kodi.bin but with Xcode it builds it in a Debug/Release subfolder. Could we just make it a phony target?